api/proto/rekor_service.proto

```diff
@@ -16,17 +16,83 @@ syntax = "proto3";
 package dev.sigstore.rekor.v2;
 
 import "google/api/annotations.proto";
+import "google/api/field_behavior.proto";
+import "google/api/httpbody.proto";
+import "logentry.proto";
+import "hashedrekord.proto";
+import "dsse.proto";
 
 option go_package = "github.com/sigstore/rekor-tiles/pkg/generated/protobuf";
 
+// A service for sigstore clients to connect to to create log entries
 service Rekor {
-    rpc Echo (StringMessage) returns (StringMessage) {
+    // Create an entry in the log
+    rpc CreateEntry (CreateEntryRequest) returns (TransparencyLogEntry) {
         option (google.api.http) = {
-            get: "/api/v2/echo"
+            post: "/api/v2/log/entries"
+            body: "*"
         };
     }
 }
 
-message StringMessage {
-  string value = 1;
+// Create a new HashedRekord or DSSE
+message CreateEntryRequest {
+    oneof spec {
+        HashedRekordRequest hashed_rekord_request = 1 [(google.api.field_behavior) = REQUIRED];
+        DSSERequest dsse_request  = 2 [(google.api.field_behavior) = REQUIRED];
+    }
+}
+
+// A service for log monitors and witnesses to audit/inspect the log
+service RekorLog {
```

Comment:
This is missing the /api/v2/checkpoint endpoint (https://docs.google.com/document/d/1qZ-lkpbQkBzV45rtemWYmT6ReqCwjTt5TbMDFLdaPyM/edit?resourcekey=0-bMAyN9EKPDvB0H3edYi_Cw&tab=t.0) - cacheability described in https://github.com/C2SP/C2SP/blob/main/tlog-tiles.md#checkpoints